modules/snooker/src/test/ts/module/ephox/snooker/test/Assertions.ts

```diff
@@ -9,18 +9,31 @@ import { RunOperationOutput, TargetElement, TargetPasteRows, TargetSelection } f
 import * as Bars from 'ephox/snooker/resize/Bars';
 import * as Bridge from 'ephox/snooker/test/Bridge';
 
-type Op<T> = (
+type OperationSingle<T> = (
   wire: ResizeWire,
   table: SugarElement,
   target: T,
   generators: Generators,
 ) => Optional<RunOperationOutput>;
 
+type OperationMultile<T> = (
```

Comment:
`Multile` isn't a word, so maybe `Multi` or `Multiple`?

Reply:
Well, it was an attempt to spell `multiple`...

Reply:
This still hasn't been fixed, so can't mark as resolved sorry @HAFRMO